tests/unit/Application/Metadata/JsonGenerator/PrivacyQuestionsMetadataGeneratorTest.php

```diff
@@ -54,9 +53,9 @@ public function test_it_can_build_privacy_question_metadata()
         $service->setPrivacyQuestions($privacyQuestions);
         $entity->setService($service);
 
-        $metadataRepository = new AttributesMetadataRepository(__DIR__ . '/../../../../../assets/Resources');
-
-        $factory = new PrivacyQuestionsMetadataGenerator($metadataRepository);
+        $factory = new PrivacyQuestionsMetadataGenerator(
+            new AttributesMetadataRepository(__DIR__ . '/../../../../../assets/Resources')
```

Comment:
The `/../../../../../` part does make it hard to read the path, but I assume there is no better option available?